dsb-integration/src/test/java/com.spectralogic.dsbrowser.integration/CloseConfirmationHandlerTest.java

```
@@ -37,44 +41,61 @@
 
     @BeforeClass
     public static void setConnection() {
         new JFXPanel();
         Platform.runLater(() -> {
-            final SavedSession savedSession = new SavedSession(SessionConstants.SESSION_NAME, SessionConstants.SESSION_PATH, SessionConstants.PORT_NO, null, new SavedCredentials(SessionConstants.ACCESS_ID, SessionConstants.SECRET_KEY), false);
-            session = createConnectionTask.createConnection(SessionModelService.setSessionModel(savedSession, false));
-            handler = new CloseConfirmationHandler(null, null, null, null, null, jobWorkers, workers);
-            final ClassLoader classLoader = CloseConfirmationHandlerTest.class.getClassLoader();
-            final URL url = classLoader.getResource(SessionConstants.LOCAL_FOLDER + SessionConstants.LOCAL_FILE);
-            if (url != null) {
-                CloseConfirmationHandlerTest.file = new File(url.getFile());
+            final SavedSession savedSession = new SavedSession(
+                    "CloseConfirmationTest",
+                    client.getConnectionDetails().getEndpoint(),
+                    "80",
+                    null,
+                    new SavedCredentials(client.getConnectionDetails().getCredentials().getClientId(), client.getConnectionDetails().getCredentials().getKey()),
+                    false,
+                    false);
+            session = createConnectionTask.createConnection(SessionModelService.setSessionModel(savedSession, false), resourceBundle, buildInfoService);
+            handler = new CloseConfirmationHandler(resourceBundle, jobWorkers, Mockito.mock(ShutdownService.class));
+            try {
+                final Path path = ResourceUtils.loadFileResource("files/");
+                if (path != null) {
+                    file = path.toFile();
+                }
+            } catch (URISyntaxException | FileNotFoundException e) {
+                e.printStackTrace();
```

If this is a legitimate error condition, then let it fall out to the junit runner instead of handling it here, and swallowing the exception.